{"name": "Проверка получения ошибки аутентификации курьера при вводе невалидных данных", "status": "passed", "description": "В тест по очереди передаются наборы данных с несуществующим логином или неверным паролем. Проверяются код и тело ответа.", "parameters": [{"name": "nonexistent_credentials", "value": "{'login': 'Eat.805', 'password': 'V%I6lJsU!4'}"}], "start": 1726902692761, "stop": 1726902693616, "uuid": "83a5a187-47b3-4efe-b384-2a03574fb4f7", "historyId": "e81e624e3035dcaca63476a2e06ad0a8", "testCaseId": "199be2f2af7455b14ed4795a4ee970da", "fullName": "tests.test_courier_login.TestCourierLogin#test_courier_login_nonexistent_data_not_found", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "DESKTOP-I4FH3MC"}, {"name": "thread", "value": "105144-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}